### PR TITLE
Removed explicitly stated thread and analogues as components from the recipes and replaced them with filament

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Also, updated filament qty in recipes where needed.

### CODE REVIEW

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -671,16 +666,13 @@
...
   "components": [
     [[ "stick", 1 ], [ "2x4", 1 ]],
     [[ "steel_chunk", 1 ], [ "steel_lump", 1 ]],
     [
-      [ "cordage_short", 2, "LIST" ],
-      [ "sinew", 40 ],
-      [ "thread", 40 ],
-      [ "plant_fibre", 40 ],
-      [ "yarn", 40 ]
+      [ "cordage_short", 2, "LIST" ]
```
Comment (Cyrano7) [inline data/json/recipes/recipe_others.json]: I thank what your trying to do here is allow for cordage OR filament but how you have the recipe now is cordage AND filament. IDK if you would necessarily need both to make a makeshift hammer.

Comment (Night-Pryanik) [reply data/json/recipes/recipe_others.json]: Fixed.

File: data/json/recipes/recipe_others.json [hunk 2]
```diff
@@ -648,20 +648,14 @@
   "skill_used": "survival",
   "time": 5000,
   "autolearn": true,
+  "using" : [ [ "cordage_short", 2 ], [ "filament", 40 ] ],
```
Comment (Cyrano7) [inline data/json/recipes/recipe_others.json]: If i'm not mistaking, this still means AND and not OR. You don't want to use "using" you should keep them down in "components".

Comment (Night-Pryanik) [reply data/json/recipes/recipe_others.json]: No, this means OR.

**Pull request merged**